src/components/Blockquote.tsx

```diff
@@ -5,24 +5,47 @@ import Link from 'next/link'
 
 import { buttonStyle, buttonLocation } from '@data'
 
-interface Logo {
+interface LogoProps {
     src: string
     alt: string
     href?: string
 }
 
-interface Link {
+interface LinkProps {
```

Comment:
Since `Logo` and `Link` aren't components, should they have this naming convention, @bretthayes ?

Reply:
I agree! No, I don't think they should have the same naming convention. I'm reconsidering the naming convention and I think we should revisit the discussion in the issue for clarification.